sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/WritableColumnVector.java

```diff
@@ -738,6 +774,12 @@ public WritableColumnVector arrayData() {
    */
   protected boolean isConstant;
 
+  /**
+   * True if this column only contains nulls. Comparing to 'isConstant' above, this doesn't require
+   * any allocation of space.
```

Comment:
```suggestion
   * True if this column only contains nulls. This means the column values never change, even
   * across resets. Comparing to 'isConstant' above, this doesn't require
   * any allocation of space.
```

Based on `reset` change.